caluma/form/migrations/0004_auto_20180827_1105.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-27 11:05
+from __future__ import unicode_literals
+
+from django.contrib.postgres.operations import HStoreExtension
+from django.db import migrations
+
+import localized_fields.fields.field
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("form", "0003_auto_20180824_1334")]
+
+    operations = [
+        # needed till https://github.com/SectorLabs/django-localized-fields/pull/55
+        # is merged
+        HStoreExtension(),
+        migrations.AlterModelOptions(
+            name="formquestion", options={"ordering": ("-sort", "id")}
+        ),
```

Comment:
Why is this in here?

Reply:
makemigrations created this as when this alteration was added makemigrations was not called (this change is actually not database relevant but makemigrations doesn't know this at time of creation)